src/panels/PatientControlPanel.jsx

```diff
@@ -18,6 +20,10 @@ class PatientControlPanel extends Component {
             width: '30px',
             height: '40px'
         };
+        this.state = {
+            isModalOpen: false
+        };
+        this.isTablet = props.isTablet;
```

Comment:
[JA1]: Is there a reason why we assign this variable to the prop, rather than just using `this.props.isTablet` throughout the component?

Reply:
Nope no reason. Fixed!